.circleci/config.yml.j2

```diff
@@ -631,8 +631,10 @@ jobs:
       - attach_workspace:
           at: {{ workspace_prefix }}
       - run:
-          name: Load archived Docker image
-          command: docker load -i {{ workspace_prefix }}/saved-images/<< parameters.image_name >>-onbuild.tar
+          name: Load archived Docker image or pull it from the registry
+          command: |
+            docker load -i {{ workspace_prefix }}/saved-images/<< parameters.image_name >>-onbuild.tar || \
+            docker pull quay.io/astronomer/<< parameters.image_name >>-onbuild
```

Comment:
I'm not sure this will actually work, as it's scanning `"<< parameters.image_name >>-onbuild"`. You'd have to tag it for this to work, and that feels a little messy?

Reply:
Refactored to something that I think will work better.